fern/definition/cloud/games/tokens.yml

```diff
@@ -12,6 +12,16 @@ service:
         game_id:
           type: uuid
       response: CreateCloudTokenResponse
+    createServiceToken:
```

Comment:
This endpoint goes by "ServiceToken" whereas the namespace endpoint goes by "TokenService". Either one should be changed for consistency